data/hai/hai reveal 6 end.txt

```diff
@@ -49,9 +49,23 @@ mission "Hai Reveal [C01] Cleanup"
 		log `Have learned that during the crisis a single ship loaded with kidnapped Hai was misplaced and abandoned. They require rescue and may indicate the presence of a loose end.`
 		conversation
-			`Arriving on Mountaintop, you see the station has seen serious renovations, including a grand ballroom, all manner of diplomatic suites, and a notable lack of movie props. Rather than actors and stage crew, you see many familiar and unfamiliar people in full diplomatic regalia. The Chancellor of the Free Worlds and President of the Republic are in their most respectable suits, Alondo and Xilin have the traditional symbols of office (though Alondo's looks like the Free Worlds have perhaps not finalized their design just yet), and dress uniforms laden with medals abound on both sides. Throughout the preliminary meetings, you notice Remington and Terry are missing.`
-			`	Between sessions, Xilin explains the Syndicate's absence. "The Syndicate is technically part of the Republic and awarded no special diplomatic status. Besides, Terry seems to have disappeared from Syndicate communications, though I won't hazard a guess as to their internal politics."`
-			`	You follow the diplomats to a room called the grand Borealis Chamber, "Where Mountains Touch The Sky," as Teeneep makes the first publicly broadcast speech from the Hai to humanity. The three heads of state announce a resolution granting safe passage to all Hai in human space, as long as they register with customs on Mountaintop before entering, and Teeneep expresses similar sentiments for humans visiting Hai space. It is announced that extensive surveillance fleets will be posted on both sides of the wormhole to ensure safety. This meeting will now turn into a multi-day summit to discuss the creation of a formal treaty.`
+			`Arriving on Mountaintop, you see the station has seen serious renovations, including a grand ballroom, all manner of diplomatic suites, and a notable lack of movie props. Rather than actors and stage crew, you see many familiar and unfamiliar people in full diplomatic regalia. The Chancellor of the Free Worlds and President of the Republic are in their most respectable suits, Alondo and Xilin have the traditional symbols of office (though Alondo's looks like the Free Worlds have perhaps not finalized their design just yet), and dress uniforms laden with medals abound on both sides. You appear to be a guest of honor, being occasionally called upon to answer only a few idle queries from people seeking clarity on one point or another - and otherwise allowed to mingle with anyone taking a break from their duties. Throughout these preliminary meetings, you notice Remington and Terry are conspicuously absent among the attendees.`
```

Comment:
```suggestion
			`Arriving on Mountaintop, you see the station has seen serious renovations including a grand ballroom, all manner of diplomatic suites, and a notable lack of movie props. Rather than actors and stage crew, you see many familiar and unfamiliar people in full diplomatic regalia, and dress uniforms laden with medals abound on all sides. The Chancellor of the Free Worlds and President of the Republic are in their most respectable suits, while Alondo and Xilin have the traditional symbols of office (though Alondo's looks like the Free Worlds have perhaps not finalized their design just yet). You appear to be a guest of honor, occasionally being called upon to answer idle queries from people seeking clarity on one point or another but otherwise left to your self. As you mingle with those taking a break from their duties throughout these preliminary meetings, you notice Remington and Terry are conspicuously not among the attendees.`
```

This is another place I think the additions are good in their own right, but they further convolute and confuse an already awkward passage.

I don't think the comma there is correct/appropriate, but I'm also not sure nothing is. This may be best as `:` after `including`, but I'll admit I'm not very strong on when those are used - just not really my style. Next change is to switch from `X/Y people, X/Y people, and objects` pattern shift in that sentence, especially since it's broken by the parenthetical aside anyway. Next sentence is unnecessarily complex I think, but also using I don't think that `- and` is used correctly. Last is more fixing of the previous, but that `absent among the attendees` also feels awkward/redundant. An alternative would be to make it just ` are conspicuously absent`.

Reply:
Adopting your suggestion with the following minor tweaks:
1. your self -> yourself
2. make it just  `are conspicuously absent.`
```suggestion
			`Arriving on Mountaintop, you see the station has seen serious renovations including a grand ballroom, all manner of diplomatic suites, and a notable lack of movie props. Rather than actors and stage crew, you see many familiar and unfamiliar people in full diplomatic regalia, and dress uniforms laden with medals abound on all sides. The Chancellor of the Free Worlds and President of the Republic are in their most respectable suits, while Alondo and Xilin have the traditional symbols of office (though Alondo's looks like the Free Worlds have perhaps not finalized their design just yet). You appear to be a guest of honor, occasionally being called upon to answer idle queries from people seeking clarity on one point or another but otherwise left to yourself. As you mingle with those taking a break from their duties throughout these preliminary meetings, you notice Remington and Terry are conspicuously absent.`
```